src/adapters/comms-adapter.ts

```diff
@@ -59,10 +59,12 @@ function createWsRoomAdapter(
             users: res.users,
             details: res.details
               .filter((room: any) => room.roomName.startsWith(roomPrefix))
+              .filter((room: any) => room.count > 0)
```

Comment:
nitpick: this can be done in the same filter (I think the filter is not optimized to to the right thing and put them together)